src/datumaro/plugins/transforms.py

```diff
@@ -1967,12 +1981,22 @@ def transform_item(self, item):
                 "Item %s: TableRow info is required for this " "transform" % (item.id,)
             )
 
+        sep_token = self._sep_token
         refined_media = self.refine_tabular_media(item) if item.media.has_data else None
-        annotation_values = {
-            key: refined_media.data[key] for key in item.annotations[0].values.keys()
-        }  # only for tabular
-        refined_annotations = [
-            ann.wrap(values=annotation_values) if isinstance(ann, Tabular) else ann
-            for ann in item.annotations
-        ]
+        refined_annotations = []
+        for ann in item.annotations:
+            if isinstance(ann, Tabular):
+                annotation_values = {
```

Comment:
You might be able to put in a kind of checker here such as
```suggestion
            if isinstance(ann, Tabular):
                if len(item.annotations) != 1:
                    msg = ...
                    raise ValueError(msg)
                annotation_values = {
```

Reply:
I added this through [afd56c5](https://github.com/openvinotoolkit/datumaro/pull/1533/commits/afd56c5ceaf32e6d778d05753e1b4ea27d7943be). Thank you for your suggestion.